test/prism_regression/assoc_splat.rb

```diff
@@ -0,0 +1,7 @@
+# typed: false
+
+{ **foo }
+
+# with other hash keys
+{ a: 1, **foo }
+{ **foo, a: 1 }
```

Comment:
Any other cases to be mindful of here? The other hash key/value pairs technically don't impact the assoc splat node, but I figured it would be reassuring to test it in a couple more contexts.